### PR TITLE
visualsource - source code visualization badge

### PR DESCRIPTION
**Pull request by ktamas77**
Link to dynamically generated source code visualization as badge (refreshing automatically)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+[![Gitter](https://badges.gitter.im/okTurtles/group-income.svg)](https://gitter.im/okTurtles/group-income) [![Build Status](https://img.shields.io/travis/okTurtles/group-income-simple/master.svg)](https://travis-ci.org/okTurtles/group-income-simple) [![Deps](https://david-dm.org/okTurtles/group-income-simple.svg)](https://david-dm.org/okTurtles/group-income-simple/#info=dependencies) [![Dev Deps](https://david-dm.org/okTurtles/group-income-simple/dev-status.svg)](https://david-dm.org/okTurtles/group-income-simple/#info=devDependencies) [![Visual Source](https://img.shields.io/badge/visual-source-orange)](http://www.visualsource.net/repo/github.com/okTurtles/group-income-simple) [![Donate](https://img.shields.io/badge/donate%20-%3D%E2%9D%A4-blue.svg)](https://okturtles.org/donate/)
```
Comment (taoeffect) [inline README.md:5]: `https:` link plz!

**Review by taoeffect — APPROVED** [review]
@ktamas77 this is awesome, thank you! Really cool project!! Just one request for the future, an HTTPS link plz!

(I'm approving this since it's easy to change myself.)

File: README.md (hunk 1 above)
Comment (ktamas77) [reply README.md:5]: of course! it meant to be https.
thank you for updating it: https://github.com/okTurtles/group-income-simple/commit/224fac56e0f88fe4d6ef4b1d3227b2ecf129171b

Comment (ktamas77) [reply README.md:5]: of course! it meant to be https.
thank you for updating it: https://github.com/okTurtles/group-income-simple/commit/224fac56e0f88fe4d6ef4b1d3227b2ecf129171b

**Pull request merged**